File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/transactions/models/TransactionsManagerModule.java
package com.fintechplatform.ui.transactions.models;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class TransactionsManagerModule {
    @Provides
    @Singleton
    TransactionsManager providesTransactionsManager() {
        return new TransactionsManager();
    }
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/transactions/di/TransactionsManagerModule.kt
package com.fintechplatform.ui.transactions.di

import com.fintechplatform.ui.transactions.models.TransactionsManager
import dagger.Module
import dagger.Provides
import javax.inject.Singleton


@Module
class TransactionsManagerModule {
    @Provides
    @Singleton
    internal fun providesTransactionsManager(): TransactionsManager {
        return TransactionsManager()
    }
}
